index.html

```diff
@@ -75,8 +75,31 @@
           <p><a class="btn btn-default" href="#" role="button">View details &raquo;</a></p>
        </div>
         <div class="col-md-4">
-          <h2>Heading</h2>
-          <p>Donec sed odio dui. Cras justo odio, dapibus ac facilisis in, egestas eget quam. Vestibulum id ligula porta felis euismod semper. Fusce dapibus, tellus ac cursus commodo, tortor mauris condimentum nibh, ut fermentum massa justo sit amet risus.</p>
+          <h2>My life is a progress meter</h2>
+          <p><i>by Fridge</i>
+            Starting off with great enthusiasm<br>
+            only to be slowed by a mutex lock<br>
+            <strong>13%</strong> installed<br>
```

Comment:
I agree with Ryan and Jiun. I think that the italicized text already adds emphasis.